integration_tests/integration/bookedtoday/arrivals/hasNoMatchAndPncOnly.spec.ts

```diff
@@ -105,4 +108,29 @@ context('No match found', () => {
       value().contains('20 September 1982')
     }
   })
+
+  it('Can process arrival', () => {
```

Comment:
Moved this test over as this now falls under this scenario - with its previous owner now falling under the isSingleMatch